### DISCUSSION TITLE
1.28 for Windows

### DISCUSSION THREAD

**Discussion by mstaver** [Lolliedieb/lolMiner-releases#518] [General]
I don't see the Windows 1.28 build... is there one?

**Comment by FaradaysInfinity** [comment]
I was looking for this as well

**Comment by Shillelagh-Hummus** [comment]
https://github.com/Lolliedieb/lolMiner-releases/releases/download/1.28/lolMiner_v1.28a_Win64.zip

**Reply by RogueTechnomancer** [reply]
The download is not starting for some reason

**Reply by Shillelagh-Hummus** [reply]
Very odd, worked fine for me.
